Hestia/app/src/main/java/hestia/UI/AddDeviceInfo.java

```diff
@@ -42,65 +48,59 @@ protected void onCreate(Bundle savedInstanceState) {
         int count = 0;
 
         for (String key : fields.keySet()) {
-            LinearLayout ll = new LinearLayout(getContext());
+            LinearLayout ll = new LinearLayout(content);
 
             // Add text
-            TextView name = new TextView(getContext());
+            TextView name = new TextView(content);
             name.setText(key);
             ll.addView(name);
 
             //Add field
-            EditText field = createEditText();
-            field.setText(fields.get(key));
-            field.setId(count);
-            if (fields.get(key) != null) {
-                field.setFocusable(false);
-                field.setClickable(false);
-            }
-            field.setLayoutParams(params);
+            EditText field = createEditText(key, params , count);
             ll.addView(field);
 
             lm.addView(ll);
             count++;
         }
         LinearLayout ll = generateButtons(params);
         lm.addView(ll);
-
     }
 
-    private EditText createEditText() {
-        final LinearLayout.LayoutParams lparams = new LinearLayout.LayoutParams(800, 150);
-        final EditText edittext = new EditText(getContext());
-        edittext.setLayoutParams(lparams);
-        edittext.setWidth(800);
-        return edittext;
+    private EditText createEditText(String key, LinearLayout.LayoutParams params, int count) {
+        final EditText field = new EditText(content);
+        field.setText(fields.get(key));
+        field.setId(count);
+        if (fields.get(key) != null) {
+            field.setFocusable(false);
+            field.setClickable(false);
+        }
+        field.setLayoutParams(params);
+        field.setWidth(800);
+        return field;
```

Comment:
It is, but we considered that it would be better for the User to have an overview of the organization and plugin name while entering the ip address (or any other relevant information).

Or would you like it to be a simple bolded text?